rotordc_website_theme/static/src/scss/rotordc.scss

```diff
@@ -196,6 +217,20 @@ a:hover {
 /*text overlay on thumbnail */
 .oe_product section {
     background-color: #fff;
+    padding: 0 15px 0 15px;
+
+    h6 {
+        margin-top: 0.5rem;
+    }
+
+    .product_availability {
+        color: grey;
+        margin-bottom: 0.5rem;
+    }
+
+    .product_price > .product_availability {
+        display: none;
+    }
```

Comment:
Is this hiding the div ? Isn't the common way of doing to add `hidden` or `invisible` attribute to the given element ?

Reply:
As we are in the "frontend", meaning the website of Odoo, we do not have access to the `invisible` options of the `<field>` or `<t t-field />` tags.

In order to not break possible child module of `website_sale_product_availability` we hide element with CSS rules.